### DISCUSSION TITLE
Swiss Dev Confederation

### DISCUSSION THREAD

**Discussion by insectengine** [quarkusio/quarkus#43249] [Events]
**Event Description:** Red Hat organizes the first Swiss Dev Confederation. Learn from engineers, practitioners & open source committers and get the latest insights into cloud native development.

**Date:** September 17, 2024
**Location:**  JED Events, Zürcherstrasse 39, Schlieren, 8952, Zurich, Switzerland
**Event Type:** In Person

https://events.redhat.com/profile/form/index.cfm?PKformID=0x11354792edc#overview

**Developer Joy == Developer Productivity (really!)**
**Speaker(s):** Holly Cummins
**Day/Time:** 11:30 - 12:00

**Crafting Intelligent Application with Quarkus - Harness the power of LLMs [Intermediate]**
**Speaker(s):** Dimitris Andreadis
**Day/Time:** 15:30 - 16:10

**Comment by oleg-nenashev** [comment]
Do you know whether the videos will be published? 
I wanted to follow-up on some Quarkus content, and to also reference two Gradle-related presentations in our channels

**Reply by insectengine** [reply]
Sorry, I don't have any direct visibility into this. @holly-cummins @dandreadis  - do either of you know this?

**Reply by holly-cummins** [reply]
My slides are here: https://hollycummins.com/productivity-equals-joy-swiss-redhat/, and @aloubyansky and @dandreadis  may also have a link for their talks. Unfortunately, I've asked and they weren't recorded.

**Reply by holly-cummins** [reply]
Ah, and everyone's slides are at https://events.redhat.com/profile/form/index.cfm?PKformID=0x1135498c29b&pkRegID=0x43874368869ae#page-table. Registration (free) is needed.

**Reply by oleg-nenashev** [reply]
Anyway, thanks a lot!
